ZenPacks/zenoss/Import4/migration.py

```diff
@@ -218,7 +219,7 @@ def exec_cmd(self, cmd, status_key=None, status_max=0, status_re=None, to_log=Tr
         while True:
             _line = proc.stdout.readline()
             if _line:
-                if to_log:
+                if to_log and re.search(to_log_re, _line):
```

Comment:
This is to filter the verbose log from mysql import. Remember that we rely on the verbose output to check the insert progress. However, each insert statement is pages of characters which we don't actually need in the log output.